x-pack/plugins/maps/public/classes/layers/tiled_vector_layer/tiled_vector_layer.test.tsx

```diff
@@ -130,13 +139,49 @@ describe('syncData', () => {
     sinon.assert.notCalled(syncContext2.stopLoading);
   });
 
+  it('Should resync when changes to syncContext', async () => {
+    const layer1: TiledVectorLayer = createLayer({}, {}, true);
```

Comment:
I am not sure I understand this test. How is layer1 used and how does it effect layer2 syncing? Is it even needed? All context from previous sync is passed in DataRequestDescriptor so I do not follow how layer1 effects anything.